multistream_test.go

```diff
@@ -665,7 +665,7 @@ func (rob *readonlyBuffer) Close() error {
 	return nil
 }
 
-func TestNegotiateFail(t *testing.T) {
+func TestNegotiatThenWriteFail(t *testing.T) {
```

Comment:
@marten-seemann highlighting this change. I'm changing this test so that it passes with the current behavior (negotiate works even if you can't write to it).

Reply:
This change goes makes this behavor more like negotiatelazy (wrt write errors). See the previous change here: https://github.com/multiformats/go-multistream/pull/85/files#diff-e59d8a202c91e1303878fa9a18fea9f6a2e3258b609bf4cc1874a17314cb26f8L679